sklearn/ensemble/tests/test_bagging.py

```diff
@@ -503,7 +502,7 @@ def test_parallel_regression():
     assert_array_almost_equal(y1, y2)
 
     ensemble = BaggingRegressor(DecisionTreeRegressor(),
-                                n_jobs=1,
+                                n_jobs=3,
```

Comment:
Although this change is small, it is not related to the title. This can be done with another PR. The main reason for creating a new PR is such that we can discuss this change independently of the rest of the changes. Secondly, less diffs makes this PR easier to merge.

Reply:
Point taken, bad habits!